src/plugins/orangehrmPerformancePlugin/Api/Model/DetailedPerformanceReviewModel.php

```diff
@@ -19,49 +19,56 @@
 
 namespace OrangeHRM\Performance\Api\Model;
 
+use OrangeHRM\Core\Api\V2\Serializer\ModelTrait;
 use OrangeHRM\Core\Api\V2\Serializer\Normalizable;
-use OrangeHRM\Core\Traits\Service\NormalizerServiceTrait;
 use OrangeHRM\Entity\PerformanceReview;
 use OrangeHRM\Performance\Traits\Service\PerformanceReviewServiceTrait;
 
 class DetailedPerformanceReviewModel implements Normalizable
 {
-    use NormalizerServiceTrait;
+    use ModelTrait {ModelTrait::toArray as modelToArray;}
```

Comment:
```suggestion
    use ModelTrait {ModelTrait::toArray as entitylToArray;}
```
shall we go like this.